.github/workflows/beekeeper.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       REPLICA: 3
       RUN_TYPE: "PR RUN"
       SETUP_CONTRACT_IMAGE_TAG: "0.2.0"
-      BEEKEEPER_BRANCH: "master"
+      BEEKEEPER_BRANCH: "topup-bug"
```

Comment:
If this branch fixes the beekeeper issue, the next steps are :
1. merge the beekeeper branch to master
2. change this line back to master